diff --git a/ddummy6.js b/ddummy6.js
index 15b0132..c7fa8be 100644
--- a/ddummy6.js
+++ b/ddummy6.js
@@ -4861,7 +4861,7 @@ function load(data,statusText,jqXHR,ctx)
 	resetTimeout();
 	var tmp = data;
 	
-	tmp = JSON.parse("(" + tmp + ")");
+	tmp = JSON.parse(tmp);
 	tmp = tmp.sess;
 	
 	if (load.arguments.length>3)	// being performed locally, not on server
@@ -4897,8 +4897,8 @@ function load2(data,statusText,	jqXHR)
 	resetTimeout();
 	
 	var tmp = data;
-	tmp = JSON.parse("(" + tmp + ")");
-	
+	tmp = JSON.parse(tmp);
+
 	switch(language)
 	{
 		case "de":
@@ -4925,7 +4925,7 @@ function dddloadfunc(data,statusText,jqXHR,context)
 	if (g_mode==1)
 	{
 		var tmp =data;
-		tmp = JSON.parse("(" + tmp + ")");
+		tmp = JSON.parse(tmp);
 //				alert(tmp.sess.deltaElapsed);
 		
 		if ((tmp.sess.status<200)&&(tmp.sess.status!=0))
@@ -5542,7 +5542,7 @@ function setOptions(optionsStr)
 	try {
 		if (optionsStr!==null)
 		{
-			var options = JSON.parse("(" + optionsStr + ")");
+			var options = JSON.parse(optionsStr);
 			document.getElementById("nsrad1").checked = options.options.ns[0]==="true";
 			document.getElementById("nsrad2").checked = options.options.ns[1]==="true";
 			document.getElementById("nsrad3").checked = options.options.ns[2]==="true";
@@ -10434,7 +10434,7 @@ function loadTraveller_1(data,statusText,jqXHR,context)
 					alert(JSON.stringify(data));
 			}
 				
-			g_travellers = JSON.parse("(" + data + ")");
+			g_travellers = JSON.parse(data);
 			
 			if ((typeof g_travellers.event.participants.pair)!="undefined")
 			{
@@ -12164,7 +12164,7 @@ function loadHands_1(data,statusText,jqXHR,context)
 		hands = data;
 	}
 	
-	hands = JSON.parse("(" + hands + ")");
+	hands = JSON.parse(hands);
     
     var saved_boards = g_hands.boards;
 	
@@ -13196,7 +13196,7 @@ function buildPage1(data,options)
 	if ((typeof g_hands.lin)!=="undefined")
 	{
 		var linstr = linToJson(g_hands.lin);
-		var linhands = JSON.parse("(" + linstr + ")");
+		var linhands = JSON.parse(linstr);
 		g_hands.boards = linhands.boards;
 	}
 	
diff --git a/functions.js b/functions.js
index 5513bfa..ececbfa 100644
--- a/functions.js
+++ b/functions.js
@@ -763,7 +763,7 @@ function startup(){
 		result.boards.push(board);
 		
 // 		document.getElementById("form1").style.display = "none";
-		buildPage(result,'{\'options\':{\'ns\':[\'true\',\'false\',\'false\'],\'ew\':[\'true\',\'false\',\'false\'],\'mk\':[\'true\',\'false\'],\'auto\':\'true\'}}');
+		buildPage(result,'{"options":{"ns":["true","false","false"],"ew":["true","false","false"],"mk":["true","false"],"auto":"true"}}');
   }
   
   function readText(file)
@@ -789,7 +789,7 @@ function startup(){
                 if (result!="")
                 {
 // 				    document.getElementById("form1").style.display = "none";
-                    buildPage(result,'{\'options\':{\'ns\':[\'true\',\'false\',\'false\'],\'ew\':[\'true\',\'false\',\'false\'],\'mk\':[\'true\',\'false\'],\'auto\':\'true\'}}');			
+                    buildPage(result,'{"options":{"ns":["true","false","false"],"ew":["true","false","false"],"mk":["true","false"],"auto":"true"}}');			
                 }
           },
           false,
@@ -814,7 +814,7 @@ function startup(){
 // 			if (result!="")
 // 			{
 // // 				document.getElementById("form1").style.display = "none";
-// 				buildPage(result,'{\'options\':{\'ns\':[\'true\',\'false\',\'false\'],\'ew\':[\'true\',\'false\',\'false\'],\'mk\':[\'true\',\'false\'],\'auto\':\'true\'}}');			
+// 				buildPage(result,'{"options":{"ns":["true","false","false"],"ew":["true","false","false"],"mk":["true","false"],"auto":"true"}}');			
 // 			}
 // 		};//end onload()
 		
@@ -865,7 +865,7 @@ function startup(){
 */
 		
 		if (result!=""){
-			buildPage(result,'{\'options\':{\'ns\':[\'true\',\'false\',\'false\'],\'ew\':[\'true\',\'false\',\'false\'],\'mk\':[\'true\',\'false\'],\'auto\':\'true\'}}');
+			buildPage(result,'{"options":{"ns":["true","false","false"],"ew":["true","false","false"],"mk":["true","false"],"auto":"true"}}');
 		}
 
 		setupGeneralHelp();
@@ -1456,7 +1456,7 @@ function processClipboardData(text)
 
 	if (isPBN || isLIN || isDLM)
 	{
-		buildPage(result,'{\'options\':{\'ns\':[\'true\',\'false\',\'false\'],\'ew\':[\'true\',\'false\',\'false\'],\'mk\':[\'true\',\'false\'],\'auto\':\'true\'}}');
+		buildPage(result,'{"options":{"ns":["true","false","false"],"ew":["true","false","false"],"mk":["true","false"],"auto":"true"}}');
 	} else {
 		switch(language)
 		{
